set-check.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 target="$1"
-secdir="${2:-../cloudfoundry/secure}"
-secfile="concourse-secrets.yml.gpg"
-secrets="${secdir}/${secfile}"
 
-if [ ! -f "${secrets}" ]
-then
-    echo -e 1>&2 "$0: Failed to find the secrets file ${secrets}\n\tPlease specify the correct directory holding \"${secfile}\"."
-    exit 1
+if test -n "${CONCOURSE_SECRETS_FILE:-}"; then
```

Comment:
We should probably error out if it's not set, otherwise we end up later with
```bash
gpg -d --no-tty "" 2> /dev/null
```
And since we eat stderr that's going to be troublesome to debug.
(We don't seem to have `set -o nounset` in this file.)